Examples/BookingSystem.AspNetCore/Stores/SellerStore.cs

```diff
@@ -22,13 +23,13 @@ public AcmeSellerStore(bool useSingleSellerMode)
         }
 
         // If the Seller is not found, simply return null to generate the correct Open Booking error
-        protected override ILegalEntity GetSeller(SellerIdComponents sellerIdComponents)
+        protected override ValueTask<ILegalEntity> GetSellerAsync(SellerIdComponents sellerIdComponents)
```

Comment:
Need the `async` keyword so .NET can do its magic on the return values
```suggestion
        protected async override ValueTask<ILegalEntity> GetSellerAsync(SellerIdComponents sellerIdComponents)
```